From a4407602566b0809eabe530ce00b433fcac1da72 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 19 Apr 2017 15:01:50 +0300
Subject: [PATCH 0658/5242] arm64: imx: Use dummy busfreq implementation

commit  40672196a41d9b45ff029d72dca154075aceced0 from
https://source.codeaurora.org/external/imx/linux-imx.git

There are no busfreq methods implemented for
ARM64, so use the dummy methods in this case.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/busfreq-imx.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/busfreq-imx.h b/include/linux/busfreq-imx.h
index d309f3d..35d57fe 100644
--- a/include/linux/busfreq-imx.h
+++ b/include/linux/busfreq-imx.h
@@ -41,7 +41,7 @@ enum bus_freq_mode {
 	BUS_FREQ_ULTRA_LOW,
 };
 
-#ifdef CONFIG_CPU_FREQ
+#if defined(CONFIG_CPU_FREQ) && !defined(CONFIG_ARM64)
 void request_bus_freq(enum bus_freq_mode mode);
 void release_bus_freq(enum bus_freq_mode mode);
 int register_busfreq_notifier(struct notifier_block *nb);
-- 
1.7.9.5

